server.go

```diff
@@ -3345,9 +3346,14 @@ func (s *Server) DiagnosticsAsRows() []*influxql.Row {
 
 		// Shard may not be local to this node.
 		if sh.store != nil {
-			shardsRow.Columns = append(shardsRow.Columns, "path")
-			shardsRow.Values[0] = append(shardsRow.Values[0], sh.store.Path())
+			if !hasPath {
+				shardsRow.Columns = append(shardsRow.Columns, "path")
+				hasPath = true
+			}
+			index := len(shardsRow.Values) - 1 // Current shardsRow.Values index
+			shardsRow.Values[index] = append(shardsRow.Values[index], sh.store.Path())
```

Comment:
Sure, before my change the server was doing something like this: 

```shards_diag
[2015-04-25 01:15:03.956810841 +0000 UTC 1 1 550 /home/marcos/.influxdb/data/shards/1 /home/marcos/.influxdb/data/shards/2] map[id:1 dataNodes:2 index:3 path:5]
[2015-04-25 01:15:03.956810841 +0000 UTC 2 1 221] map[index:3 path:5 id:1 dataNodes:2]
```
After my change is doing this


```shards_diag
[2015-04-25 01:15:03.956810841 +0000 UTC 1 1 550 /home/marcos/.influxdb/data/shards/1] map[id:1 dataNodes:2 index:3 path:4]
[2015-04-25 01:15:03.956810841 +0000 UTC 2 1 221 /home/marcos/.influxdb/data/shards/2] map[index:3 path:4 id:1 dataNodes:2]
```
My understanding by looking at the code is that the `shard_diag` row should contain one row per shard, and previously it was appending all the shards in the first row